nautobot/virtualization/models.py

```diff
@@ -391,7 +392,7 @@ def site_id(self):
     "statuses",
     "webhooks",
 )
-class VMInterface(BaseModel, BaseInterface, CustomFieldModel):
+class VMInterface(BaseModel, BaseInterface, CustomFieldModel, NotesMixin):
```

Comment:
Huh. Is VMInterface not a ChangeLoggedModel? Looks like none of the device component models are change-logged either. That's probably not ideal.

Reply:
Agreed, probably outside my scope, but I will see what I can do.

Reply:
I think this is why they do not have ChangedLoggedModel, in testing, they do have the changes logged.
```
class ComponentModel(BaseModel, CustomFieldModel, RelationshipModel, NotesMixin):
    """
    An abstract model inherited by any model which has a parent Device.
    """

...

    def to_objectchange(self, action):
        # Annotate the parent Device
        try:
            device = self.device
        except ObjectDoesNotExist:
            # The parent Device has already been deleted
            device = None

        return ObjectChange(
            changed_object=self,
            object_repr=str(self),
            action=action,
            object_data=serialize_object(self),
            object_data_v2=serialize_object_v2(self),
            related_object=device,
        )
```